src/Components/Employee.js

```diff
@@ -1,15 +1,19 @@
 import PetList from "./PetList";
 import "./Employee.css";
+import { useState } from "react";
 
-export const Employee = () => {
+export const Employee = ({ employee }) => {
+  const [showPets, setShowPets] = useState(false)
+  console.log(showPets)
   return (
     <article className="employee">
-      <h3>Staff Member Name</h3>
-      <h4>Staff Member Title</h4>
-      <button>Show Pets</button>
-      <PetList />
+      <h3>{employee.prefix} {employee.firstName} {employee.lastName}{employee.postfix && `, ${employee.postfix}`}</h3>
+      <h4>{employee.title}</h4>
+      <button onClick={() => { setShowPets(!showPets) }}>Show Pets</button>
+      <PetList employeeId={employee.id} />
```

Comment:
you have the logic for toggling `showPets` on the click but you're missing the conditional rendering logic. Once you fix that you pass al the tests